x/posts/keeper/common_test.go

```diff
@@ -62,20 +60,20 @@ type TestData struct {
 	postCreationDate       time.Time
 	postEndPollDate        time.Time
 	postEndPollDateExpired time.Time
-	answers                types2.PollAnswers
-	registeredReaction     types2.RegisteredReaction
-	post                   types2.Post
-	subspace               types3.Subspace
+	answers                types.PollAnswers
+	registeredReaction     types.RegisteredReaction
+	post                   types.Post
+	subspace               subspacestypes.Subspace
 }
 
 func (suite *KeeperTestSuite) SetupTest() {
 	// Define the store keys
-	keys := sdk.NewMemoryStoreKeys(types2.StoreKey, paramstypes.StoreKey, profilestypes.StoreKey, types3.StoreKey,
+	keys := sdk.NewMemoryStoreKeys(types.StoreKey, paramstypes.StoreKey, profilestypes.StoreKey, subspacestypes.StoreKey,
```

Comment:
```suggestion
	keys := sdk.NewKVStoreKeys(types.StoreKey, paramstypes.StoreKey, profilestypes.StoreKey, subspacestypes.StoreKey,
```
I think it should be KVStoreKeys, instead.